packages/webview_flutter/webview_flutter_android/android/src/main/java/io/flutter/plugins/webviewflutter/ProxyApiRegistrar.java

```diff
@@ -56,6 +56,27 @@ void logError(String tag, Throwable exception) {
             + Log.getStackTraceString(exception));
   }
 
+  /** Creates an exception when the `unknown` enum value is passed to a host method. */
+  @NonNull
+  IllegalArgumentException createUnknownEnumException(@NonNull Object enumValue) {
+    return new IllegalArgumentException(enumValue + " doesn't represent a native value.");
+  }
+
+  /** Creates the error message when a method is called on an unsupported version. */
+  @NonNull
+  String createUnsupportedVersionMessage(
+      @NonNull String method, @NonNull String versionRequirements) {
+    return method + " requires " + versionRequirements + ".";
+  }
+
+  /** Creates an exception when a method is called on an unsupported version. */
+  @NonNull
+  UnsupportedOperationException createUnsupportedVersionError(
```

Comment:
Is this actually used anywhere? I was wondering why both this and the `Message` version above existed, and I only found usage of that one.